Modules/@babylonjs/react-native-iosandroid/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@babylonjs/react-native-iosandroid",
+  "title": "React Native Babylon for iOS and Android",
+  "version": "0.0.1",
+  "description": "Babylon Native integration into React Native",
+  "main": "index.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/babylonjs/BabylonReactNative.git",
+    "baseUrl": "https://github.com/babylonjs/BabylonReactNative"
+  },
+  "keywords": [
+    "react-native"
+  ],
+  "author": {
+    "name": "Your Name",
+    "email": "yourname@email.com"
+  },
+  "homepage": "https://github.com/BabylonJS/BabylonReactNative#readme",
+  "license": "MIT",
+  "licenseFilename": "LICENSE",
+  "readmeFilename": "README.md",
+  "dependencies": {
+    "base-64": "^0.1.0",
+    "semver": "^7.3.2"
+  },
+  "peerDependencies": {
+    "@babylonjs/core": "^5.0.0-rc.4",
+    "@babylonjs/react-native": "version",
+    "react": ">=16.13.1",
+    "react-native": ">=0.63.1",
+    "react-native-permissions": "^3.0.0"
```

Comment:
Maybe similar comment here, about these being transitive dependencies through @babylonjs/react-native (I think that is the only place we actually have TypeScript dependencies on these). I think for react-native we still need the dependency at build time, but maybe that means react-native should be a devDependency for this module?

Reply:
I don't have a clear picture of 'good' dependency chain here. Feel free to update the PR or do another one if it's not good.